bot/classes/command.js

```diff
@@ -36,6 +36,11 @@ class Command extends CommandModel {
      * @returns True if the command is triggered by the discordMessage
      */
     triggeredBy(discordMessage, loweredCaseMessage, prefixes) {
+        if (!global.discordClient._ready) {
```

Comment:
I don't think we need that check here, because if the client isn't ready, we won't receive any command anyway